src/OmniSharp.Host/Startup.cs

```diff
@@ -36,18 +38,21 @@ public Startup(IOmniSharpEnvironment env)
 
             var configBuilder = new ConfigurationBuilder()
                 .SetBasePath(AppContext.BaseDirectory)
-                .AddJsonFile("config.json", optional: true)
+                .AddJsonFile(OmniSharpConstants.OmnisharpConfigFile, optional: true)
                 .AddEnvironmentVariables();
 
             if (env.OtherArgs?.Length > 0)
             {
                 configBuilder.AddCommandLine(env.OtherArgs);
             }
 
+            // Use the global omnisharp config if there's any in the shared path
+            configBuilder.CreateAndAddGlobalOptionsFile(env);
```

Comment:
I agree, I was a bit on the fence with this one. The idea was to be able to say: "got here, and edit the `omnisharp.json` file to have global shared settings", rather than having to say "please create it".

However, you are right, an empty file is a bit silly. 

I think we should still create the folder though, it would be somewhat awkward to have to manually create folders that OmniSharp needs.